typescript/integrations.d.ts

```diff
@@ -197,6 +198,17 @@ export type IntegrationScheduling<S = SavedSecret> = IntegrationBase & {
   }
 }
 
+export type IntegrationNylasGrant<S = SavedSecret> = {
```

Comment:
Good point, does the grantId itself need to be a secret?

Reply:
Josh-ninion: no, since you still need to have some form of Auth to use it. I think it's equivalent to an `accountId` from V2 and we never encrypted that, so I'd say it's also fine for that reason